README.md

```diff
@@ -0,0 +1,26 @@
+# EPICS IOC for uTCA crate management
+
+[![Continuous Integration Status](https://github.com/lnls-dig/ipmi-mgr-epics-ioc/actions/workflows/build.yml/badge.svg)](https://github.com/lnls-dig/ipmi-mgr-epics-ioc/actions)
+
+This repository contains the EPICS Input/Output Controller (IOC) used at LNLS
+based on [ipmiComm](https://github.com/slac-epics-modules/ipmiComm) support module.
```

Comment:
```suggestion
This repository contains the EPICS Input/Output Controller (IOC) used at LNLS for monitoring and controlling MTCA crates,
based on the [ipmiComm](https://github.com/slac-epics-modules/ipmiComm) support module.
```

Reply:
what about change `monitoring and controlling` for `managing`?